src/api/spec/factories/users.rb

```diff
@@ -2,7 +2,7 @@
   factory :user do
     email { Faker::Internet.email }
     realname { Faker::Name.name }
-    login { Faker::Internet.user_name(nil, %w(_)) }
+    sequence(:login){|n| "#{Faker::Internet.user_name(nil, %w(_))}#{n}" }
```

Comment:
Do we still need faker here? We could just switch to something like "user_factory_#{n}"